docs: update docs to reflect node v18 is current (#14467)
OpenTrons/opentrons-api: DEV_SETUP.md

nvs --version
```

> Now we can use `nvs` to install the currently required Node.js version set in `.nvmrc`. We think it's easiest if you switch on `auto` mode which will make sure the correct version of Node.js is used any time we're in the `opentrons` project directory. ^[If you don't want to use `nvs` auto mode you'll have to manually run `use` or `install` when you want to work on the project.]

Reviewer: ```suggestion
Now we can use `nvs` to install the currently required Node.js version set in `.nvmrc`. We enable `auto` mode to use the correct version of Node.js any time we're in the `opentrons` project directory. Without `auto` mode, we would have to manually run `use` or `install` each time we work on the project.
```
Markdown footnotes don't render inline in GitHub, which is where most users will read this. So I would remove that syntax. Plus some light prose edits.

Reviewer 2: I think we'd like to remove "mode" from use `nvs` auto mode since it is a command or switching the word from mode to command would be better.

https://github.com/jasongin/nvs?tab=readme-ov-file#command-reference

Reviewer: How about:
```
The `auto` command selects the correct version of Node.js any time we're in the `opentrons` project directory.
Without `auto`, we would have to manually run `use` or `install` each time we work on the project.
```

Reviewer 2: I think `the correct version` -> `the long-term support (LTS) version` would be accurate.
what do you think about that, @ecormany ?

Reviewer: I used "correct" just to mean "it will work" 😁 

I don't think we have to be more precise here. (And I don't know enough about how `nvs` behaves to be more precise.)

Reviewer 2: For nvs/nvm, we set 18 in `.nvmrc` so if someone uses `nvs auto` or `nvm use`, nvs/nvm will install LTS version of nodejs. Right now it is v 18.19.0.

Author: @ecormany @koji any final thoughts on this as to the final version of the wording? I don't have strong feelings so happy to follow whatever you settle on as best

Reviewer 2: Hi @strangemonad 

If you adopted the following Ed's suggestion, that would be great. 
```
The `auto` command selects the correct version of Node.js any time we're in the `opentrons` project directory.
Without `auto`, we would have to manually run `use` or `install` each time we work on the project.
```
